localstack/utils/bootstrap.py

```diff
@@ -130,7 +131,27 @@ def get_main_container_name():
     return MAIN_CONTAINER_NAME_CACHED
 
 
-def get_server_version():
+def get_image_hash():
+    image_name = get_docker_image_to_start()
+    image_id = DOCKER_CLIENT.inspect_image(image_name)["Id"]
+    if image_id.startswith("sha256:"):
+        return image_id.split(":")[1][:12]
+    return image_id
+
+
+def get_image_environment_variable(env_name: str):
```

Comment:
type hints for the output type (`Optional[str]`) would be cool :-) same for `get_image_hash()` and the other methods here